_data/committers.yml

```
@@ -453,7 +462,7 @@
   affiliation: ByteDance
 - name: Xuwei Fu
   role: Committer
-  alias: mwish
+  alias: maplefu
   affiliation: Unknown
```

```suggestion
  affiliation: Singdata
```

Xuwei is my colleague :)